src/utils/email.js

```diff
@@ -51,40 +51,45 @@ async function initEmail (smtpUrl = AppConstants.SMTP_URL) {
  * @param {string} recipient
  * @param {string} subject
  * @param {string} html
- * @returns {Promise<SMTPTransport.SentMessageInfo>} <Promise>
+ *
+ * @returns {Promise<SMTPTransport.SentMessageInfo>}
  */
-function sendEmail (recipient, subject, html) {
+async function sendEmail (recipient, subject, html) {
```

Comment:
I also turned this into an `async function` so we don't need to do quite so much wrapping.